service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -323,8 +332,20 @@ public ResponseEntity<RunSetStateResponse> postRunSet(RunSetRequest request) {
         new RunSetStateResponse().runSetId(runSetId).runs(runStateResponseList).state(runSetState);
 
     captureResponseMetrics(response);
-    bardService.logRunSetEvent(request, userToken);
-
+    if (bardServerConfiguration.enabled()) {
+      List<String> workflowIds =
+          runStateResponseList.stream()
+              .map(RunStateResponse::getEngineId)
+              .filter(Objects::nonNull)
+              .toList();
+      GithubMethodDetails githubMethodDetails = null;
+      if (methodSourceEnum.equals(PostMethodRequest.MethodSourceEnum.GITHUB)) {
+        githubMethodDetails =
+            githubMethodDetailsDao.getMethodSourceDetails(methodVersion.getMethodId());
+      }
+      bardService.logRunSetEvent(
+          request, methodVersion, githubMethodDetails, workflowIds, userToken);
```

Comment:
FYI this ties bard in to only work in "raw github import" workflows. Is that intended? Or would metrics for a dockstore workflow also be important?

Reply:
As a side note: the changes in #279 would presumably make this interface easier too, since you won't need an extra parameter just for the githubMethodDetails

Reply:
CBAS will log to Bard in both types of workflows, but if the source is github it will include additional details like the org and repo. I'll add in the dockstore info too!